sdk/Managed/src/Microsoft.WindowsAzure.MobileServices/Table/Sync/MobileServiceSyncContext.cs

```diff
@@ -303,6 +303,28 @@ public Task CancelAndUpdateItemAsync(MobileServiceTableOperationError error, JOb
             });
         }
 
+        public Task UpdateOperationAsync(MobileServiceTableOperationError error, JObject item)
+        {
+            string itemId = error.Item.Value<string>(MobileServiceSystemColumns.Id);
```

Comment:
Is error.item guaranteed to be populated, I thought it was optional.  (For instance a synced delete could be done on an item not present locally, etc)